content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -465,12 +465,18 @@ kubectl apply --server-side --field-manager=my-manager [--dry-run=server]
 
 ## API Endpoint
 
-With the Server Side Apply feature enabled, the `PATCH` endpoint accepts the
+With the Server-Side Apply feature enabled, the `PATCH` endpoint accepts the
 additional `application/apply-patch+yaml` content type. Users of Server Side
```

Comment:
```suggestion
additional `application/apply-patch+yaml` content type. Users of Server-Side
```

Reply:
nice catch, my very simple search-and-replace didn't catch this case ;-)

Reply:
Actually I found another instance!

Reply:
Nice. That is one reason I like to start and end each sentence on its own line. One's searches are not foiled by the end of a line.